localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ light_group:0 "Light"
+ light_group_adj:0 "Light"
+ light_group_adherent:0 "Light Follower"
+ light_group_adherent_plural:0 "Light Followers"
+ light_group_desc:0 "A group of religions worshipping a force known as the Light. They say its positive energies, residing in every living being, in every heart and soul, is everywhere, binding all sentient beings together as one. Those who reject this light inside are nothing more than heretics."
+ 
+ holy_light:0 "Holy Light"
+ holy_light_adj:0 "Holy Light"
+ holy_light_adherent:0 "Light Follower"
+ holy_light_adherent_plural:0 "Light Followers"
+ holy_light_desc:0 "The main organisation of the Holy Light has been the Church of the Holy Light. Most humans, dwarves and even some high elves follow the path of the Church. It is a faith that advocates purity, goodness, kindness, and compassion in the face of adversity."
+ 
+ scarlet_light:0 "Scarlet Light"
+ scarlet_light_adj:0 "Scarlet Light"
+ scarlet_light_adherent:0 "Scarlet Follower"
+ scarlet_light_adherent_plural:0 "Scarlet Followers"
+ scarlet_light_desc:0 "A heretical, militant offshoot of the Church of the Holy Light. They are intolerant of all other confessions, regardless of alliance or affiliation. They believe that any and all unholy creatures must be destroyed by a cleansing flame."
+ 
+ naaru:0 "Naarism"
+ naaru_adj:0 "Naaric"
+ naaru_adherent:0 "Naarist"
+ naaru_adherent_plural:0 "Naarists"
+ naaru_desc:0 "The Light manifests itself as a golden, shining light energy. The naaru, enigmatic beings composed of shards of fractured light taught the use of the Light to the draenei and other enlightened races, but others have come into the study and practice of Light-based magic on their own accord."
+ 
+ lightbound:0 "Lightbound"
+ lightbound_adj:0 "Lightbound"
+ lightbound_adherent:0 "Lightbound"
+ lightbound_adherent_plural:0 "Lightbound"
+ lightbound_desc:0 "A heretical offshoot of the naaru teachings, the adherents to the Lightbound doctrine are dedicated to spread the influence of the Light wherever they possibly can and convert the unbelievers by any means necessary, even forcibly."
+ 
+ loa_group:0 "Loa"
+ loa_group_adj:0 "Loa"
+ loa_group_adherent:0 "Loa Follower"
+ loa_group_adherent_plural:0 "Loa Followers"
+ loa_group_desc:0 "Loa is the name given by trolls to beings they worship such as the Wild Gods or spirits. This large group of cults originated in ancient times when the first mortals started encountering magical creatures and deifying them."
+ 
+ zandism:0 "Zan'atali"
+ zandism_adj:0 "Zan'atali"
+ zandism_adherent:0 "Zan'atali"
+ zandism_adherent_plural:0 "Zan'atali"
+ zandism_desc:0 "A pantheon of Loa founded by the Zandalari trolls. This is one of the most ancient pantheons. However, it is renowned for its adaptability. $zandism_adherent_plural$ respect all other Loa in order to maintain friendly ties with other confessions."
```

Again, confessions is a little confusing.
```suggestion
 zandism_desc:0 "A pantheon of Loa founded by the Zandalari trolls. This is one of the most ancient pantheons. However, it is renowned for its adaptability. $zandism_adherent_plural$ respect all other Loa in order to maintain friendly ties with other Loa sects."
```